extensions/wikipedia/src/show-details-page.tsx

```diff
@@ -0,0 +1,31 @@
+import { ActionPanel, Detail, OpenInBrowserAction, CopyToClipboardAction } from "@raycast/api";
+import { encodeTitle } from "./wikipedia";
+
+function ShowDetailsPage(props: { title: string; extract: string | undefined; description: string | undefined }) {
+  const markdown = `
+  
+  # ${props.title}
+  
+  ###### ${props.description ? props.description : ""}
+
+  ---
+
+  ${props.extract} [read more](https://wikipedia.org/wiki/${encodeTitle(props.title)})
+
+  `;
+
+  return (
+    <Detail
+      navigationTitle={props.title}
+      markdown={markdown}
+      actions={
+        <ActionPanel>
+          <OpenInBrowserAction url={`https://wikipedia.org/wiki/${encodeTitle(props.title)}`} />
+          <CopyToClipboardAction title="Copy URL" content={`https://wikipedia.org/wiki/${encodeTitle(props.title)}`} />
```

Comment:
To match the `search-page` behaviour I would like to have `⌘+.` or `⌘+Shift+C` for copying the URL.

Reply:
I have added `⌘` + `.` shortcut in `Show Details` page